storage_service/locations/api/resources.py

```diff
@@ -412,6 +436,8 @@ class PackageResource(ModelResource):
     current_full_path = fields.CharField(attribute='full_path', readonly=True)
     related_packages = fields.ManyToManyField('self', 'related_packages', null=True)
 
+    default_location_regex = re.compile(r'\/api\/v2\/location\/default\/(?P<purpose>[A-Z]{2})\/?')
```

Comment:
We're going to use this regex later under the `hydrate_current_location` method`.